### ISSUE TITLE
Ignore package-lock.json

### ISSUE THREAD

**Issue by Jimgitsit** [gk646/CodeStats#12]
Would it be possible to add an option to ignore package-lock.json?

**Comment by gk646** [comment]
Hey, thanks for the suggestion.
Sure, ill add it to one of the existing options and push an update.

**Comment by gk646** [comment]
Blacklisting or whitelisting individual based on name isnt a feature current.y
I didnt really think of that. Currently you can only exclude directories or whole file types.
The problem is that checking each file name with regex can get slow.
But this needs more work than i though. 
Might come sometime in the future.

**Comment by Jimgitsit** [comment]
Would be nice if we could explicitly ignore just the `package-lock.json` file, because it can get huge and throws off the counts by a lot.

**Comment by gk646** [comment]
Yep i understand that. It is a missing feature. Ill add a regex based file exclude field simialr to the directories.

**Comment by gk646** [comment]
Sorry for the delay. The new update is already live.
Now the regex also applies to all files (should have been from the start i guess).
To exclude a file in the root just add the full name as regex:
`packagelock.json`
Thanks for waiting
